packages/main/src/Table.js

```diff
@@ -50,6 +50,28 @@ const metadata = {
 		},
 	},
 	properties: /** @lends sap.ui.webcomponents.main.Table.prototype */ {
+
+		/**
+		 * Defines the text that will be displayed when there is no data and <code>showNoData</code> is present.
+		 *
+		 * @type {String}
+		 * @defaultvalue: ""
+		 * @public
+		 */
+		noDataText: {
+			type: String,
+		},
+
+		/**
+		 * Defines if the value of <code>no-data-text</code> will be diplayed when there is no rows present in the table.
```

Comment:
just minor one, in the JS Doc we refer to the property (not attribute) name, so ca you replace "no-data-text" with "noDataText"